src/plugins/vis_type_vega/public/vega_inspector/components/inspector_data_grid.tsx

```diff
@@ -110,26 +110,26 @@ export const InspectorDataGrid = ({ columns, data, dataGridAriaLabel }: Inspecto
   }, [gridData, pagination]);
 
   // Resize
-  const [columnsWidth, setColumnsWidth] = useState<Record<string, number>>({});
+  const [columnWidths, setColumnWidths] = useState<Record<string, number>>({});
```

Comment:
Was this name change just for consistency? Or was it actually necessary?

Reply:
Yeah, I think it is a better name. It is not a must change. But since column width (uiState) is not saved in saved object and won't hurt current customer usage or need any migration, I think it is effortless to adopt a better name for consistency.